components/email_aliases/email_aliases_service_unittest.cc

```diff
@@ -345,3 +345,337 @@ TEST_F(EmailAliasesServiceTimingTest, VerifyResult_StopsAfterMaxDuration) {
 // checking cancellation of polling, etc.
 
 }  // namespace email_aliases
+namespace email_aliases {
+
+class EmailAliasesAPITest : public ::testing::Test {
+ protected:
+  EmailAliasesAPITest() {
+    feature_list_.InitAndEnableFeature(email_aliases::kEmailAliases);
+  }
+
+  void SetUp() override {
+    service_ = std::make_unique<EmailAliasesService>(
+        url_loader_factory_.GetSafeWeakWrapper());
+
+    mojo::PendingRemote<email_aliases::mojom::EmailAliasesServiceObserver>
+        remote;
+    observer_.BindReceiver(remote.InitWithNewPipeAndPassReceiver());
+    service_->AddObserver(std::move(remote));
+  }
+
+  // Minimal observer capturing alias updates.
+  class AliasObserver
+      : public email_aliases::mojom::EmailAliasesServiceObserver {
+   public:
+    void OnAuthStateChanged(email_aliases::mojom::AuthStatePtr) override {}
+    void OnAliasesUpdated(
+        std::vector<email_aliases::mojom::AliasPtr> aliases) override {
+      ++alias_updates;
+      last_aliases = std::move(aliases);
+    }
+    bool WaitForAliasUpdateCount(size_t count) {
+      return base::test::RunUntil([&]() { return alias_updates >= count; });
+    }
+
+    size_t alias_updates = 0;
+    std::vector<email_aliases::mojom::AliasPtr> last_aliases;
```

Comment:
Class fields are expected to be private or protected:

https://google.github.io/styleguide/cppguide.html#Access_Control

Reply:
fixed